examples/plugins/wrapstore/cli/simplestore.go

```diff
@@ -0,0 +1,111 @@
+// +build evm
+
+package main
+
+import (
+	"fmt"
+	"github.com/loomnetwork/go-loom"
+	"github.com/loomnetwork/go-loom/auth"
+	"github.com/loomnetwork/go-loom/client"
+	"github.com/loomnetwork/go-loom/examples/plugins/wrapstore/types"
+	"github.com/spf13/cobra"
+	"golang.org/x/crypto/ed25519"
+	"log"
+	"os"
+)
+
+var RootCmd = &cobra.Command{
+	Use:   "simplestore",
+	Short: "SimpeStore DAppChain",
+}
+
+type persistFlags struct {
+	WriteUri        string `json:"wrjiteuri"`
+	ReadUri         string `json:"readuri"`
+	ContractHexAddr string `json:"contractaddr"`
+	ChainId         string `json:"chainid"`
+}
+
+func main() {
+	var flags persistFlags
+	RootCmd.PersistentFlags().StringVarP(&flags.WriteUri, "write", "w", "http://localhost:46657", "URI for sending txs")
+	RootCmd.PersistentFlags().StringVarP(&flags.ReadUri, "read", "r", "http://localhost:9999", "URI for quering app state")
+	RootCmd.PersistentFlags().StringVarP(&flags.ContractHexAddr, "contract", "c", "0x005B17864f3adbF53b1384F2E6f2120c6652F779", "contract address")
+	RootCmd.PersistentFlags().StringVarP(&flags.ChainId, "chainId", "i", "default", "chain ID")
+
+	getCmd := &cobra.Command{
+		Use:   "get",
+		Short: "get the value from the store",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			result, err := GetValueCmd(flags.ChainId, flags.WriteUri, flags.ReadUri, flags.ContractHexAddr)
+			if err == nil {
+				log.Printf("{ Out: '%d' }", result)
+				fmt.Println("output:", result)
+			}
+			return err
+		},
+	}
+
+	var value int
+	setCmd := &cobra.Command{
+		Use:   "set",
+		Short: "set the value in the store",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return SetValueCmd(flags.ChainId, flags.WriteUri, flags.ReadUri, flags.ContractHexAddr, value)
+		},
+	}
+	setCmd.Flags().IntVarP(&value, "value", "v", 0, "value to set in store")
+	RootCmd.AddCommand(
+		getCmd,
+		setCmd,
+	)
+	err := RootCmd.Execute()
+	if err != nil {
+		fmt.Println(err)
+		os.Exit(1)
+	}
+}
+
+func GetValueCmd(chainId, writeUri, readUri, contractHexAddr string) (int64, error) {
+	rpcClient := client.NewDAppChainRPCClient(chainId, writeUri, readUri)
+	contractAddr, err := loom.LocalAddressFromHexString(contractHexAddr)
+	if err != nil {
+		return 0, err
+	}
+	contract := client.NewContract(rpcClient, contractAddr, "wrapstore")
+
+	dummy := &types.Dummy{}
+	result := &types.WrapValue{}
+	// NOTE: usually you shouldn't generate a new key pair for every tx, but this is just an example...
+	_, priv, err := ed25519.GenerateKey(nil)
```

Comment:
Can we extract these key gen to a separate command, and then ask the user to run it at least once first? Then just load it from a file. Would be a better canonical example